fontir/src/variations.rs

```diff
@@ -191,6 +191,8 @@ impl VariationModel {
         {
             let master_influences = &self.delta_weights[model_idx];
             let mut deltas = Vec::new();
+            deltas.reserve(points.len());
```

Comment:
try `Vec::with_capacity(points.len())` instead of `Vec::new` instead of this. It's a good habit in general anytime you know the collection length ahead of time :)

Reply:
Done